### DISCUSSION TITLE
Adding a terraform provisioning

### DISCUSSION THREAD

**Discussion by irish1986** [techno-tim/k3s-ansible#330] [Questions]
I've been working on a fairly generic terraform provisioning for k3s using telemate/proxmox provider.

Given the nature of ansible for configuration and terraform for provisioning, should a provisioning be welcome to this repo?

This would increase repeatability and be fully IaC. What do you think?

**Comment by irish1986** [comment]
So I am back from summer vacation, any desire for this ? Should I start writting up a PR or does this kind of feature undesired for this repo ?

**Reply by timothystewart6** [reply]
I like this idea but also want to be sure it fits into the core of what this repo is trying to do (help you quickly spin up a kubernetes cluster).  Can you give a brief overview of how it will work?

* changes to the repo
* how to install/configure
* provisioning
* deprovisioning

I just want to be sure that this fits without making breaking changes and that it is optional.
